webapp/src/main/java/com/box/l10n/mojito/okapi/XLIFFWriter.java

```diff
@@ -489,6 +489,7 @@ public void writeTextUnit(ITextUnit tu, String phaseName) {
             params.getPlaceholderMode(),
             params.getIncludeCodeAttrs(),
             params.getIncludeIts(),
+            false,
```

Comment:
What is this new boolean parameter?

Reply:
it's named as: isTarget. In the new version of okapi, that parameter is used to modify the result of that method
<img width="1155" alt="Screenshot 2024-07-01 at 10 43 05 AM" src="https://github.com/pinterest/mojito/assets/40587669/e1e36184-07a9-46c1-8e8c-3f56cb651661">
we get the same output with isTarget = false

Reply:
As the comment, it's for supporting XLIFF 2.0